Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,27 @@
-import React from 'react';
-import './Button.css';
+import React from "react";
+import "./Button.css";
+
+const NumberButton = props => {
+  // console.log(props);
+  return (
+    <div>
+      <button className="clear">{props.num[0]}</button>
+      <br />
+      <button className="numBtn">{props.num[1]}</button>
+      <button className="numBtn">{props.num[2]}</button>
+      <button className="numBtn">{props.num[3]}</button>
+      <br />
+      <button className="numBtn">{props.num[4]}</button>
+      <button className="numBtn">{props.num[5]}</button>
+      <button className="numBtn">{props.num[6]}</button>
+      <br />
+      <button className="numBtn">{props.num[7]}</button>
+      <button className="numBtn">{props.num[8]}</button>
+      <button className="numBtn">{props.num[9]}</button>
+      <br />
```

Comment:
if you use flex and responsive units/percents you can avoid using these breaks in your html